cmd/ls.go

```diff
@@ -22,7 +22,7 @@ import (
 	"text/tabwriter"
 
 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```

Comment:
What's the reason for this? Please avoid named imports unless absolutely necessary.